.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,9 @@
+## Enhancement Description
+Please summarize your proposals.
+
+* Proposal: [a link to the corresponding pull request]()
```

Comment:
@style95 did you intend for this heading to be a link to the PR - wasn't clear to me.

```suggestion
* Proposal: [a link to the corresponding pull request if it exists already]()
```